manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -0,0 +1,275 @@
+          properties:
+            outputs:
+              description: Outputs are named destinations for log messages.
+              items:
+                description: Output defines a destination for log messages.
+                properties:
+                  elasticsearch:
+                    type: object
+                  fluent:
+                    type: object
+                  insecure:
+                    description: Insecure must be true for intentionally insecure
+                      outputs. Has no function other than a marker to help avoid configuration
+                      mistakes.
+                    type: boolean
+                  name:
+                    description: Name used to refer to the output from a `pipeline`.
+                    type: string
+                  secretRef:
+                    description: "SecretRef refers to a `secret` object for secure
+                      communication. \n Client-authenticated TLS is enabled if the
+                      secret contains keys `tls.crt`, `tls.key` and `ca.crt`. Output
+                      types with password authentication will use keys `password`
+                      and `username`, not the exposed 'username@password' part of
+                      the `url`."
+                    properties:
+                      name:
+                        description: Name is unique within a namespace to reference
+                          a secret resource.
+                        type: string
+                      namespace:
+                        description: Namespace defines the space within which the
+                          secret name must be unique.
+                        type: string
+                    type: object
+                  syslog:
+                    description: 'Syslog provides optional extra properties for `type:
+                      syslog`'
+                    properties:
+                      facility:
+                        description: "Facility to set on outgoing syslog records.
+                          \n The string value may contain an integer or one of these
+                          names:     kernel user mail daemon auth syslog lpr news
+                          \    uucp cron authpriv ftp ntp security console cron     local0
+                          local1 local2 local3 local4 local5 local6 local7"
+                        type: string
+                      payloadKey:
+                        description: PayloadKey specifies record field to use as payload.
+                        type: string
+                      severity:
+                        description: "Severity to set on outgoing syslog records.
+                          \n The string value may contain an integer or one of these
+                          names:     emergency alert critical error warning notice
+                          informational debug"
+                        type: string
+                      tagKey:
```

Comment:
as per rfc 5424, the TAG has been split into APP-NAME, PROCID, and MSGID.  should we expose these?

Reply:
Yep, the Tag/Payload/Trim stuff should be aligned with your implementation. I'll leave that for you to update since you're closer to the details.